### PR TITLE
Feature/gulp tasks

### PR DESCRIPTION
**Pull request by Crevax**
#32 All that's missing is a task for FTP'ing the files, which can be achieved using https://www.npmjs.com/package/vinyl-ftp once we have that info.

### CODE REVIEW

**Comment by Crevax** [comment]
@jacebrowning @joel6653

File: Makefile [hunk 1]
```diff
@@ -80,6 +80,11 @@ ci: check test tests
 .PHONY: run
 run: geojson
 
+.PHONY: web-build
+web-build: .depends-dev gulp-build
+gulp-build:
```
Comment (jacebrowning) [inline Makefile]: Is `gulp-build` the name of the output directly?

Comment (Crevax) [inline Makefile]: No, I just thought that's how I needed to set things up to ensure the dev dependencies were installed before running gulp (and to not run the dependency install each time we want to run a gulp build). Right now the gulp build task creates two file: app.min.js and styles.min.css.

Comment (jacebrowning) [inline Makefile]: I think if you delete `gulp-build` and line 85 it will work as expected.

Comment (Crevax) [inline Makefile]: I'll test that out after work today.

**Comment by joel6653** [comment]
perhaps at next meeting you (@crevax) can help me understand how to use this

**Comment by jacebrowning** [comment]
I don't think we have a need for this anymore.

**Pull request closed without merging**